cmd/process-agent/main_common.go

```diff
@@ -163,6 +164,11 @@ func runAgent(exit chan struct{}) {
 	agentVersion, _ := version.Agent()
 	log.Infof("running version: %s", agentVersion.GetNumberAndPre())
 
+	fmt.Println("helloo")
+
+	// Only logging when process agent is running
+	misconfig.ToLog(string(loggerName))
```

Comment:
I would probably not use the `loggerName` as it's not guaranteed to stay the same. IMO probably the best bet here is to define an enum in the `misconfig` package. Something like this:

```go
type AgentType int

const (
  CoreAgent = iota,
  ProcessAgent,
)
```

That way we can change
```go
misconfig.ToLog(string(loggerName))
```
to
```go
misconfig.ToLog(misconfig.ProcessAgent)
```